index.md

```diff
@@ -3,37 +3,45 @@ title: Sysdig Helm Charts
 description: Repository for Sysdig's Helm charts.
 ---
 
-## Add this charts repository to your local Helm
+Helm is an open-source package manager for Kubernetes applications. Using a helm chart you can describe, install, configure, and upgrade applications automatically.
 
-To start using our charts, first add this _charts repository_ using the `helm` command line tool:
+To simplify your deployment experience, Sysdig provides helm charts to install and configure Sysdig installation components in your Kubernetes environment.
+
+
+## Sysdig Chart Repository
+
+The Sysdig Chart Repository includes the following charts:
+
+| Charts          | Description |
+| :---------------| ----------- |
+|[`sysdig-deploy`](/charts/sysdig-deploy)    | Deploys the following components in your Kubernetes cluster:<br>`kspm-collector`:  Enables collecting and sendin Kubernetes resource manifests to be evaluated against compliance policies. The scan results are displayed by Sysdig Secure's Actionable Compliance.<br>`node-analyzer`: Enables  Image Analyzer, Benchmarks, and Host Scanning features.<br>`rapid-response`:  Allows designated advance users to investigate and troubleshoot events from a remote shell.<br>`agent`: Processes syscall events and metrics, creates [capture](https://docs.sysdig.com/en/docs/sysdig-secure/investigate/captures/#captures) files, and performs auditing and compliance. |
+|[`admission-controller`](/charts/admission-controller) | Deploys the [Sysdig Admission Controller](https://docs.sysdig.com/en/docs/sysdig-secure/scanning/admission-controller/) in your Kubernetes cluster to enable image scanning and audit logging. |
+|[`cloud-connector`](/charts/cloud-connector) | Deploys the [Sysdig Cloud Connector](https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/) in your Kubernetes cluster to enable threat-detection and image scanning. |
```

Comment:
for the descriptions link of Cloud-Connector i would specifically point to this link
https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/#helm-chart-based-feature-limited